api/types/types.proto

```diff
@@ -3625,4 +3624,13 @@ message ConnectionDiagnosticV1 {
 message ConnectionDiagnosticSpecV1 {
     bool Success = 1 [ (gogoproto.jsontag) = "success" ];
```

Comment:
Does this represent an overall success/failure of the connection test?

Reply:
Yes.